src/extensibility/node/ExtensionManagerDomain.js

```diff
@@ -272,7 +273,7 @@ function _cmdInstall(packagePath, destinationDirectory, options, callback, _doUp
         // If the extension is already there, we signal to the front end that it's already installed
         // unless the front end has signaled an intent to update.
         if (hasLegacyPackage || fs.existsSync(installDirectory) || fs.existsSync(systemInstallDirectory)) {
-            if (_doUpdate) {
+            if (_doUpdate === true) {
```

Comment:
I'm assuming the intent was to prevent similar bugs in the future: if you accidentally pass an extra callback arg in this position, it's truthy but not `=== true`.

That said, silently acting like the param was `false` doesn't necessarily mean we're preventing the mistake from causign a bug.  Might be nice to have something like `console.assert(_doUpdate === undefined || typeof _doUpdate === "boolean")` at the top if we're truly worried about this problem happening again.

Or the other option would be to add a unit test that covers this, so that regardless of what breaks it we'd find out via a failing test :-)